src/ModelStore.ts

```diff
@@ -92,7 +92,14 @@ export class ModelStore {
             if (data && data[Constants.ITEMS_PROP]) {
                 const localData = clone(newData);
                 const items = data[Constants.ITEMS_PROP] || {};
-
+                const oldData = items[itemKey];
+                
+                for (const property in oldData) {
+                    if (!localData.value[property]) {
+                        localData.value[property] = "";
```

Comment:
Just checking -
is the value of each property expected to be of type string only ?

Reply:
Good question
I don't think every value will be string 
Intent here is to add missing property with empty value 
Since null == undefined is true this could cause issues in some cases where its not checked properly
Is there another option ?

Reply:
true for `==` but not for `===`

Reply:
> Since null == undefined is true this could cause issues in some cases where its not checked properly

Shouldn't we then fix those checks instead to use `===` and not `==` ?

Reply:
Okay, understood after a discussion. Resolving this conversation.